data/mods/ssb/abilities.js

```diff
@@ -389,6 +389,17 @@ let BattleAbilities = {
 			}
 		},
 	},
+	// inactive
+	souleater: {
+		desc: "Attacking moves heal the user 33% of damage dealt.",
+		shortDesc: "Attacking moves heal the user 33% of damage dealt.",
+		id: "souleater",
+		name: "Soul Eater",
+		isNonstandard: "Custom",
+		onModifyMove(move) {
+			move.drain = [1, 3];
```

Comment:
Dose this interact well with status moves?

Reply:
Yes, i tested it and it doesn’t do anything harmful or beneficial regarding status moves